themes/default/layouts/page/pricing.html

```diff
@@ -1467,7 +1467,7 @@ <h2 class="text-center">Compare editions</h2>
                 <div class="compare-plans-table-row-container">
                     <div class="compare-plans-table-row-item">
                         <span
-                            ><a class="compare-table-link" href="{{ relref . " /docs/intro/pulumi-service/organization-access-tokens/" }}"
```

Comment:
[This PR](https://github.com/pulumi/pulumi-hugo/pull/2242) has the trailing `/` but passes CI: https://github.com/pulumi/pulumi-hugo/actions/runs/3483734518/jobs/5827540138#step:7:60 (that's the place that the downstream CI fails). Is there a switch used there that's not used here, because here pages are expected to be missing, perhaps?

Reply:
The relref here should fail linting too, since https://github.com/pulumi/pulumi-hugo/pull/2071, but is in layouts/ so isn't linted :-/